4/README.md

```diff
@@ -0,0 +1,207 @@
+
+- Update the MongoDB and Tendermint version(s) to the latest supported version:
+
+  - mongo:3.4.13
+  - tendermint/tendermint:0.13
+
+- Deprecate the redundant Dockerfiles and only keep two:
+  - Dockerfile
+  - Dockerfile-dev
+
+- Remove `compose/bigchaindb-server`, `compose-bigchaindb-driver`, and `compose/travis`.
+
+- [IF NEEDED]: Introduce `bigchaindb/tools` directory for tools required for development i.e.
+  - `bigchaindb-driver`
+    - Python driver to interact and transact with the BigchainDB server.
+  - `curl-client`
+    - To verify if tendermint service is up i.e. making a curl call to `/abci_query` endpoint and BigchainDB root endpoint.
+  - `nginx`
+    - For hosting docs
+
+### Technical challenges
+N/A
+
+### API impact
+N/A
+
+### Security impact
+N/A
+
+### Performance impact
+N/A
+
+### End user impact
+This will impact and introduce ease for end users i.e. users deploying BigchainDB for development and testing.
+
+**Build**
+
+```
+# For development/testing and CI
+docker-compose build
+
+# For development with tools e.g. bigchaindb-driver,curl-client, nginx(hosting docs),
+# everything else is inherited from docker-compose.yml
+docker-compose -f docker-compose.dev.yml build
+```
+
+**Run**
+
+```
+# For development/testing and CI
+docker-compose up bdb
+```
+
+```
+# For development
+docker-compose -f docker-compose.dev.yml up bdb
+```
+
+**Run tests**
+```
+# Without coverage
+docker-compose -f docker-compose.yml run --rm --no-deps bdb pytest -v
+
+OR
+
+# Without coverage and using dev file
+docker-compose -f docker-compose.dev.yml run --rm --no-deps bdb pytest -v
+
+# With coverage
+docker-compose -f docker-compose.yml run --rm --no-deps bdb pytest -v --cov=bigchaindb
+
+OR
+
+# With coverage and using dev file
+docker-compose -f docker-compose.dev.yml run --rm --no-deps bdb pytest -v --cov=bigchaindb
+```
+
+**Build docs**
+```
+docker-compose -f docker-compose.yml run --rm --no-deps bdb make -C docs/server html
+
+OR
+
+# with dev file
+docker-compose -f docker-compose.dev.yml run --rm --no-deps bdb make -C docs/server html
+```
+
+
+### Deployment impact
+One impact on the deployment strategy is covered with earlier in the [End user impact section](#end-user-impact) of this document.
+
+Another deployment impact this change will have is with CI, we will need to update the CI scripts @`bigchaindb/.ci` to use `docker-compose.yml` instead of `docker-compose.travis.yml`.
+
+### Documentation impact
+Following documents will be impacted:
+
+- [CONTRIBUTING.md](https://github.com/bigchaindb/bigchaindb/blob/tendermint/CONTRIBUTING.md)
+- [Run BigchainDB with Docker](https://docs.bigchaindb.com/projects/server/en/tendermint/appendices/run-with-docker.html)
+- [tests/README.d](https://github.com/bigchaindb/bigchaindb/blob/tendermint/tests/README.md)
+- [TENDERMINT_INTEGRATION.rst](https://github.com/bigchaindb/bigchaindb/blob/tendermint/TENDERMINT_INTEGRATION.rst)
+
```

Comment:
+ The new set of docs added to `docs/contributing/` by pull request https://github.com/bigchaindb/bigchaindb/pull/2119

Reply:
Yes you could add it, since it will some day point to docs about using Docker Compose to run a local node for dev/test.